build.gradle

```diff
@@ -193,6 +198,17 @@ task fatJar(type: Jar) {
     with jar
 }
 
+shadowJar {
```

Comment:
It's so that guava can be relocated to avoid conflicts with Hadoop's version. (The dataflow SDK also relocates guava in the same way BTW.) The result is basically the same as the fatJar, so fatJar could probably be replaced with shadowJar at some point.